controllers/prometheusrule_controller.go

```diff
@@ -263,7 +263,10 @@ func prometheusInnerRuleToCoralogixAlert(prometheusRule prometheus.Rule) coralog
 		notificationPeriod = defaultCoralogixNotificationPeriod
 	}
 
-	timeWindow := prometheusAlertForToCoralogixPromqlAlertTimeWindow[prometheusRule.For]
+	timeWindow, ok := prometheusAlertForToCoralogixPromqlAlertTimeWindow[prometheusRule.For]
```

Comment:
Ideally users should be able to configure the default options of the operator, through flags

Reply:
Since Prometheus default value is 0 this means as soon as the expression gives us some results it will fire an alert, I would say we may stay as close as possible to this behavior and use the default for as 1m